src/cinder/Triangulate.cpp

```diff
@@ -102,28 +110,70 @@ void Triangulator::addPolyLine( const PolyLine2f &polyLine )
 void Triangulator::addPolyLine( const vec2 *points, size_t numPoints )
 {
 	if( numPoints > 0 )
-		tessAddContour( mTess.get(), 2, &points, sizeof(vec2), (int)numPoints );
+		tessAddContour( mTess.get(), 2, points, sizeof(vec2), (int)numPoints );
+}
+
+void Triangulator::addPolyLine( const PolyLine3f &polyLine )
+{
+	if( polyLine.size() > 0 )
+		tessAddContour( mTess.get(), 3, &polyLine.getPoints()[0], sizeof(vec3), (int)polyLine.size() );
```

Comment:
Nope, that also seems more logical to me (probably pre c++11 code when vector didn't have the `data()` member function). I'd usually also not write `sizeof(vec3)`, but `sizeof(*polyLine.getPoints().data())` as this makes the code more generic and less prone to refactoring erors (but admittedly also more ugly)